src/pages/Admin/Venue/AnnouncementOptions/AnnouncementOptions.scss

```
@@ -0,0 +1,19 @@
+@import "scss/constants";
+
+.AnnouncementOptions {
+  display: flex;
+  justify-content: space-between;
+  padding-top: $spacing--lg;
+
+  &__left-side {
+    font-size: $font-size--lg;
+    margin-top: $spacing--xs;
+  }
+
+  &.AnnouncementOptions__edit-button {
```

if we use `ButtonNG` here, can we decrease the specificity here?
```suggestion
  &__edit-button {
```

---

done ed799a1